doc/source/theory/multiphases/cfd_dem/resolved_cfd-dem.rst

```diff
@@ -0,0 +1,5 @@
+===========================
+Resolved CFD-DEM Coupling
```

Comment:
```suggestion
Resolved CFD-DEM
```

Reply:
Agreed, CFD-DEM is by definition a coupling